lib/run-in-terminal.coffee

```diff
@@ -230,7 +248,10 @@ module.exports =
 
         atom.contextMenu.add(menu)
 
-    tree_start_terminal_here: (run) ->
+    tree_start_terminal_here: (run, par) ->
```

Comment:
`par` should be replaced with `ask_parameters` here and below, since there is no reason to make short unreadable names.